xclim/data/fr.json

```
@@ -705,23 +704,23 @@
     "abstract": "Calcul de la magnitude et la direction de la vitesse du vent à partir des deux composantes ouest-est et sud-nord."
   },
   "WIND_SPEED_FROM_VECTOR.sfcWind": {
-    "long_name": "Vitesse du vent de surface",
+    "long_name": "Vitesse du vent près de la surface",
     "description": "Magnitude de la vitesse du vent calculée à partir des deux composantes uas et vas."
   },
   "WIND_SPEED_FROM_VECTOR.sfcWindfromdir": {
-    "long_name": "Direction de provenance du vent de surface",
+    "long_name": "Vent près de la surface en provenance d'une certaine direction",
     "description": "Direction de provenance du vent calculée à partir des deux composantes uas et vas. Les vents du nord ont une direction de 360° et les vents de moins de {calm_wind_thresh} ont une direction de 0°."
   },
   "WIND_VECTOR_FROM_SPEED": {
     "title": "Vitesse cartésienne du vent à partir de la magnitude et direction de provenance.",
     "abstract": "Calcul des deux composantes ouest-est et nord-sud du vent à partir de la magnitude et la direction de provenance de sa vitesse."
   },
   "WIND_VECTOR_FROM_SPEED.uas": {
-    "long_name": "Vent d'ouest de surface",
+    "long_name": "Vent d'ouest près de la surface",
     "description": "Vitesse d'ouest du vent de surface calculé à partir de la norme de sa vitesse et de sa direction de provenance."
```

```suggestion
    "description": "Vitesse du vent d'ouest près de la surface calculé à partir de la vitesse et de la direction du vent."
```
J'ai besoin d'autres avis là-dessus.

---

c'est le problème avec vent de surface ?